### PR TITLE
[doc]unify Nebula Graph name

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Changes in this pr:
1. Unify all nebula names to **Nebula Graph**. THIS IS THE ONLY NAME FORMAT IN DOCS.
2. Change the default partition number to 100. #1330

### CODE REVIEW

File: docs/doc-tools/README.md [hunk 1]
```diff
@@ -2,49 +2,50 @@
...
 
+## Step One: Merge
 
-
-## Step One: Merge 
 Use the provided script `merge-all.py` to merge all the markdown doc files into one.
 
 ## Step Two: Generate TOC
+
 Use pandoc to generated TOC (table of content) for the file you just merged. First off, you should make sure pandoc has installed on your machine. Open your terminal and run the command:
```
Comment (wilsonyou) [inline docs/doc-tools/README.md]: First off -> firstly

Comment (wilsonyou) [inline docs/doc-tools/README.md]: Firstly, you should make sure that pandoc has been installed on your computer. Open your terminal and run the command:

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Deploy Cluster
 
-This section provides an introduction to deploy `Nebula` cluster.
+This section provides an introduction to deploy `Nebula Graph` cluster.
 
 ## Download and install package
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md]: Download and Install Package

File: docs/manual-EN/3.build-develop-and-administration/2.develop-and-interface/kv-interfaces.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ auto resp = std::move(future).get()
 
 ### Processing returned results
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/2.develop-and-interface/kv-interfaces.md]: Returned Results

File: docs/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker.md [hunk 4]
```diff
@@ -29,15 +29,15 @@ docker> make
 docker> make install
 ```
 
-## Run nebula service
+## Run Nebula Graph service
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker.md]: Services

File: docs/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Building with Docker container
 
-Nebula has provided a docker image with the whole compiling environment [vesoft/nebula-dev](https://hub.docker.com/r/vesoft/nebula-dev), which will make it possible to change source code locally, build and debug within the container. Performing the following steps to start quick development:
+Nebula Graph has provided a docker image with the whole compiling environment [vesoft/nebula-dev](https://hub.docker.com/r/vesoft/nebula-dev), which will make it possible to change source code locally, build and debug within the container. Performing the following steps to start quick development:
 
 ## Pull image from Docker Hub
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker.md]: Image

File: docs/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Building with Docker container
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker.md]: Container

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 ## Prerequisite
 
-Nebula is developed based C++14, so it requires a compiler supporting C++14 features.
+Nebula Graph is developed based C++14, so it requires a compiler supporting C++14 features.
 
 ### Supported system version
```
Comment (wilsonyou) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: Please check all the words in headings

Comment (Amber1990Zhang) [reply docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: Will open a new pr to take care of all the headings.

**Pull request merged**